debian/zabbix-check-sshkey.examples

```diff
@@ -0,0 +1 @@
+examples/zabbix-check-sshkey-template.xml
```

Comment:
`dh_installexamples: error: Cannot find (any matches for) "examples/zabbix-check-sshkey-template.xml" (tried in .)`